src/publish-vsix.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Publish individual built-in VS Code extensions to an
+ * Open VSX registry (default: open-vsx.org) . It is 
+ * assumed that the extensions to be published are present
+ * in directory "dist" at the root of this repo.
+ * 
+ * The publishing of the extensions is delegated to `ovsx`,
+ * which uses the following environment variables to know
+ * to which registry to publish-to and what personal 
+ * authentication token to use to authenticate:
+ *  OVSX_REGISTRY_URL, OVSX_PAT
+ */
+// @ts-check
+const fs = require('fs')
+const os = require('os');
+const yargs = require('yargs');
+const { root, dist, run } = require('./paths.js');
+
+const { tag } = yargs.option('tag', {
+    choices: ['latest', 'next']
+}).demandOption('tag').argv;
+
+(async () => {
+    if (tag === 'next') {
+        console.error("Open VSX does not support publishing 'next' versions at this time");
```

Comment:
@spoenemann  I do not think we have a way yet to have `latest` and `next` versions co-habitate on open-svx and being selectively consumable in a Theia product? 

I did a little research and though requested for a long time this is not supported on the Marketplace, either. So MS extensions that have `insiders` versions do not have a good mechanism to do that. For example `ms-vscode.cpptools` has its own update mechanism that pulls an `insiders` version of itself from their GH release, and subsequent updates I think too, if a preference to that effect is enabled.

Reply:
Yes the MS Marketplace has no concept of version tags, only a `preview` flag. See my comment above: going for the npm way would give us more flexibility, but it also means the UI would become more complex.